packages/schematics/src/collection/ngrx/files/__directory__/__fileName__.facade.spec.ts__tmpl__

```diff
@@ -0,0 +1,140 @@
+import { NgModule } from '@angular/core';
+import {TestBed, async, tick, fakeAsync} from '@angular/core/testing';
+
+import {Observable, combineLatest } from 'rxjs';
+import {take} from 'rxjs/operators';
+
+import { EffectsModule } from '@ngrx/effects';
+import {StoreModule, Store} from '@ngrx/store';
+
+import { NxModule } from '@nrwl/nx';
+import {DataPersistence} from '@nrwl/nx';
+
+import { <%= className %>Effects } from './<%= fileName %>.effects';
+import { <%= className %>Facade } from './<%= fileName %>.facade';
+
+import { <%= propertyName %>Query } from './<%= fileName %>.selectors';
+import { Load<%= className %>, <%= className %>Loaded } from './<%= fileName %>.actions';
+import { <%= className %>State, <%= className %>, initialState, <%= propertyName %>Reducer } from './<%= fileName %>.reducer';
+
+interface TestSchema {
+  '<%= propertyName %>' : <%= className %>State
+}
+
+const hasLength = (expected) => (actual) => expect(actual.length).toEqual(expected);
+const isLoaded = (expected) => (actual) => expect(actual).toBe(expected);
+const confirmThat = (source$,observer) => source$.pipe(take(1)).subscribe(observer);
+
+describe('<%= className %>Facade', () => {
+  let facade: <%= className %>Facade;
+  let store: Store<TestSchema>;
+  let create<%= className %>;
+
+  beforeEach(async(() => {
+    create<%= className %> = ( id:string, name = '' ): <%= className %> => ({
+       id,
+       name: name || `name-${id}`
+    });
+  }));
+
+  describe('used in application root module', () => {
+
+    beforeEach(async(() => {
+      @NgModule({
+        imports: [
+          NxModule.forRoot(),
+          StoreModule.forRoot(
+            {'<%= propertyName %>': <%= propertyName %>Reducer},
+            { initialState : {
+                '<%= propertyName %>': initialState
+              },
+              metaReducers : []
+            }),
+          EffectsModule.forRoot([<%= className %>Effects]),
+        ],
+        providers: [<%= className %>Effects, <%= className %>Facade]
+      })
+      class RootModule {}
+      TestBed.configureTestingModule({ imports: [RootModule] });
+
+      store = TestBed.get(Store);
+      facade = TestBed.get(<%= className %>Facade);
+    }));
+
+    it('loadAll() should return empty list with loaded == true', fakeAsync(() => {
+      confirmThat(facade.all<%= className %>$, hasLength(0));
```

Comment:
I think all the tests should be written in Jasmine style (`expect(something).to...`). 

Nx defines readAll and readFirst. The generated specs should use those. If they aren't usable, we need to change them rather than adding local utilities in spec files.